frontend/src/components/layout/ContentGroup.vue

```diff
@@ -5,10 +5,15 @@ Displays the content wrapped inside a card.
 <template>
   <v-sheet class="ec-content-group mb-8 mt-2">
     <slot name="title">
-      <div v-if="title" class="ec-content-group__title py-1 subtitle-1">
+      <component
+        :is="titleTag"
```

Comment:
I think you didn't use the `titleTag` property anywhere. Is this change still needed?

Reply:
It is needed for a11y. Currently if it is used inside ContentCard the usage is correct. But the nesting could also require a h3